Dockerfile

```diff
@@ -1 +1,51 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/centos/base-image:java8
+ENV PENTAHO_CE_VERSION=8.1.0.0-365
+ENV Mysql_Connector_Version=8.0.24
+ENV Mariadb_Connector_Version=2.2.6
+ENV Arkcase_Pre_Auth_Version=4-1.1.1
+ARG RESOURCE_PATH="artifacts"
 
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_pentaho \
+      MAINTAINER="Armedia LLC"
+
+#-----------PENTAHO SETUP-------------------------------------
+RUN mkdir -p /home/pentaho/data/pentaho &&\
+    mkdir -p /home/pentaho/app/pentaho && \
+    mkdir -p /home/pentaho/tmp/pentaho && \
+    useradd --system --user-group pentaho
+
+#------------PENTAHO CE------------------------------------------
+
+#Copying pentaho ce 
+COPY ${RESOURCE_PATH}/pentaho-server-ce-${PENTAHO_CE_VERSION}.zip /home/pentaho/app/pentaho
+#Installing unzip in centos
+RUN yum install -y unzip && \
+    yum install -y mysql && \
```

Comment:
Why are you installing `mysql` here?

Reply:
Also, please run `yum clean all` after you installed all your packages

Reply:
Mysql install only mysql client and added yum clean all

Reply:
@dheeraj-armedia Is Pentaho accessing a MySQL database? If yes, is it possible to install the MySQL client libraries instead of the client executable?